src/components/PostComment.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { postComment } from "../../utils";
+import "../css/PostComment.css";
+
+const PostComment = ({ article_id, setComments }) => {
```

Comment:
It might be nice to add some descriptions as to when/why posting a comment failed -  some sort of visual feedback always helps